src/IW/Db/Issue.hs

```diff
@@ -0,0 +1,35 @@
+{-# LANGUAGE QuasiQuotes #-}
+
+-- | SQL queries to work with the @users@ table.
+
+module IW.Db.Issue
+       ( getIssues
+       , getIssueById
+       , getIssuesByLabel
+       ) where
+
+import IW.App (WithError)
+import IW.Core.Issue (Issue)
+import IW.Db.Functions (WithDb, asSingleRow, query, queryRaw)
+
+getIssues :: (WithDb env m, WithError m) => m [Issue]
+getIssues = queryRaw [sql|
+    SELECT *
+    FROM issues
+|]
+
+getIssueById :: (WithDb env m, WithError m) => Int -> m Issue
```

Comment:
I think you can create issue about introducing `newtype Id` with phantom type parameter and use it instead of `Int` where you need `Id` in a separate PR (after this PR is merged).

Reply:
I will do that.